tests/cypress/e2e/features/requests_page.js

```diff
@@ -0,0 +1,302 @@
+            };
+            cy.goToTaskList();
+            cy.createAnnotationTask(
+                task.name,
+                task.label,
+                task.attrName,
+                task.textDefaultValue,
+                archiveName,
+                task.multiAttrParams,
+                null,
+                task.forProject,
+                task.attachToProject,
+                task.projectName,
+            );
+
+            cy.goToTaskList();
+            cy.openTask(taskName);
+            cy.url().then((taskUrl) => {
+                data.taskID = Number(taskUrl.split('/').slice(-1)[0].split('?')[0]);
+                cy.getJobIDFromIdx(0).then((jobID) => {
+                    cy.headlessCreateObjects([rectanglePayload], jobID);
+                });
+            });
+        });
+    });
+
+    after(() => {
+        cy.contains('.cvat-header-button', 'Cloud Storages').should('be.visible').click();
+        cy.url().should('include', '/cloudstorages');
+        cy.deleteCloudStorage(cloudStorageData.displayName);
+
+        cy.headlessDeleteProject(data.projectID);
+    });
+
+    beforeEach(() => {
+        cy.visit('/tasks');
+        cy.get('.cvat-spinner').should('not.exist');
+    });
+
+    describe('Requests page', () => {
+        it('Creating a task creates a request. Correct task can be opened.', () => {
+            cy.visit('/requests');
+            cy.get('.cvat-spinner').should('not.exist');
+            checkRequestStatus('Create Task', () => {
+                cy.contains('Finished').should('exist');
+            });
+        });
+
+        it('Creating a task creates a request. Incorrect task can not be opened.', () => {
+            const defaultAttrValue = 'Requests attr';
+            const multiAttrParams = false;
+            const advancedConfigurationParams = false;
+            const forProject = false;
+            const attachToProject = false;
+            const expectedResult = 'fail';
+
+            cy.createAnnotationTask(
+                taskName,
+                mainLabelName,
+                attrName,
+                defaultAttrValue,
+                badArchiveName,
+                multiAttrParams,
+                advancedConfigurationParams,
+                forProject,
+                attachToProject,
+                projectName,
+                expectedResult,
+            );
+            checkRequestStatus('Create Task', () => {
+                cy.get('.cvat-request-item-progress-failed').should('exist');
+            }, { resourceLink: '' });
+        });
+
+        it('Export creates a request. Task can be opened from request. Export can be downloaded after page reload.', () => {
+            cy.openTask(taskName);
+            const exportParams = {
+                type: 'annotations',
+                format: exportFormat,
+                archiveCustomName: annotationsArchiveNameLocal,
+            };
+            cy.exportTask(exportParams);
```

Comment:
As we do not re-use these parameters, this option would be shorter

```suggestion
            cy.exportTask({
                type: 'annotations',
                format: exportFormat,
                archiveCustomName: annotationsArchiveNameLocal,
            });
```